constructor/header.sh

```diff
@@ -213,6 +213,57 @@ then
     fi
 #endif
 
+#if ppc64le
+    if [ "$(uname -m)" != "ppc64le" ]; then
+        printf "WARNING:\\n"
+        printf "    Your machine hardware does not to appear be Power8 (little endian), \\n"
```

Comment:
Can we also have a way to skip this check? If `binfmt_misc` supports the architecture of the installer then I would not like to be forced to interactively confirm this, also what about when using `--force`? That usually means "no user interaction", should this abort automatically in this case, then do we need `--really-force` to indicate we don't care that `uname` is mismatched?

Reply:
All of these checks are skipped if -b is specified.